projects/modular_llm/playground.py

```diff
@@ -87,20 +87,19 @@ class ConversationNoTemplate(Conversation):
 
 
 def main():
+    from mttl.config import ExpertConfig
     from mttl.datamodule.utils import get_tokenizer_with_args
-    from mttl.models.expert_config import ExpertConfig
     from mttl.models.expert_model import MultiExpertModel
 
     setup_autocomplete()
     setup_logging()
 
-    config = ExpertConfig.parse(
-        raise_error=False,
+    config = MultiExpertModel.parse(
         c="./configs/wiki-mmlu/phi-2_flan_v3.json",
         extra_kwargs=dict(k.split("=") for k in sys.argv[1:]),
     )
     tokenizer = get_tokenizer_with_args(config.model, "gpt", "left", "left", True)
-    model = MultiExpertModel(**vars(config), tokenizer=tokenizer).to("cuda")
+    model = MultiExpertModel(**config.asdict()).to("cuda")
```

Comment:
does `config` have a pointer to `tokenizer` ? or are we not passing the tokenizer anymore

Reply:
we are not passing tokenizer anymore, afaik it's not needed